### ISSUE TITLE
Problem with resize with two grids

### ISSUE THREAD

**Issue by cjpgaspar** [gridstack/gridstack.js#546]
Hi, 
When i am using two grids and want to transfer widgets between them, the new widget gets 2 divs with the classes "ui-resizable-handle ui-resizable-se ui-icon ui-icon-gripsmall-diagonal-se" and because of that, it cannot be resizable even though the grid is resizable. 
I correct this by adding:
if (el.children()[2]) {
     el[0].removeChild(el[0].lastChild);
}

in GridStack.prototype._prepareElementsByNode after
 this.dd
            .draggable(el, {
                start: onStartMoving,
                stop: onEndMoving,
                drag: dragOrResize
            })
            .resizable(el,{
                start: onStartMoving,
                stop: onEndMoving,
                resize: dragOrResize
            });

I don't think this is the correct way, but it solve my problem.

**Comment by ambienthack** [comment]
I suggest adding 
`$(ui.draggable).resizable('destroy');`
before 
`var el = $(ui.draggable).clone(false);` in gridstack.js

like this

```
                .on(self.container, 'drop', function(event, ui) {
                    self.placeholder.detach();
                    var node = $(ui.draggable).data('_gridstack_node');
                    node._grid = self;
                    $(ui.draggable).resizable('destroy');
                    var el = $(ui.draggable).clone(false);
                    el.data('_gridstack_node', node);
                    $(ui.draggable).remove();
                    node.el = el;
                    self.placeholder.hide();
                    el
                        .attr('data-gs-x', node.x)
                        .attr('data-gs-y', node.y)
                        .attr('data-gs-width', node.width)
                        .attr('data-gs-height', node.height)
                        .addClass(self.opts.itemClass)
                        .removeAttr('style')
                        .enableSelection()
                        .removeData('draggable')
                        .removeClass('ui-draggable ui-draggable-dragging ui-draggable-disabled')
                        .unbind('drag', onDrag);
                    self.container.append(el);
                    self._prepareElementsByNode(el, node);
                    self._updateContainerHeight();
                    self._triggerChangeEvent();

                    self.grid.endUpdate();
                });
```

**Comment by radiolips** [comment]
This is resolved in released `v0.4.0`.
